examples/provider/bid/audience-network/README.md

```diff
@@ -0,0 +1,46 @@
+# audience-network-bid-delegate
+
+BidProvider for Audience Network
+
+## Usage
+
+```html
```

Comment:
@elljoh - can you check you were signed in to facebook.com in the same browser session, and that you were using a mobile user agent, please?